docs: Fix broken values.yaml links (#715)
kyma-project/telemetry-manager: docs/user/integration/opentelemetry-demo/README.md

helm upgrade --version 0.26.1 --install --create-namespace -n $K8S_NAMESPACE $HELM_OTEL_RELEASE open-telemetry/opentelemetry-demo -f https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/opentelemetry-demo/values.yaml
```

> The command above uses a prepared [values.yaml](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/opentelemetry-demo/values.yaml) provided in this `opentelemetry-demo` folder, which contains customized settings deviating from the default settings. Alternatively, you could create your own `values.yaml` file and adjust the command. The customizations in the prepared `values.yaml` cover the following areas:

Reviewer: ```suggestion
The previous command uses the[values.yaml](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/opentelemetry-demo/values.yaml) provided in this `opentelemetry-demo` folder, which contains customized settings deviating from the default settings. The customizations in the provided `values.yaml` cover the following areas:
```

Reviewer: I'd put this in line 100:

`Alternatively, you can create your own `values.yaml` file and adjust the command.`
